File: client_unity/Assets/scripts/model/MResponse.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using LitJson;

public class MResponse {
	public int code;
	public string msg;
	public JsonData data;

	public MResponse(JsonData resJson) {
		code = (int)resJson ["code"];
		msg = (string)resJson ["msg"];
		if (resJson.ContainsKey("data")) {
			data = resJson ["data"];
		}
	}

	public bool isOk () {
		return code > 0;
	}
}

File: client_cocos/assets/Script/protocol/MResponse.js
var MResponse = function (data) {
    this.data = data['data'];
    this.code = data['code'];
    this.msg = data['msg'];
}

module.exports = MResponse;


MResponse.prototype.hasError = function() {
    return this.code <= 0;
}

MResponse.prototype.isOK = function() {
    return this.code > 0;
}